x-pack/plugins/cloud_integrations/cloud_data_migration/README.md

```diff
@@ -0,0 +1,3 @@
+# Cloud Data Migration
```

Comment:
I don't have much context on the `cloud_integrations` directory in Kibana. Do you know if there are any guidelines on what should live under here?

Reply:
As far as I can tell it seems to be all cloud related implementations. e.g. Gainsight, DriftChat, FullStory, Cloud links e.t.c